spec/entities/configurations/kalibro_configuration_spec.rb

```diff
@@ -17,6 +17,14 @@
 require 'spec_helper'
 
 describe KalibroClient::Entities::Configurations::KalibroConfiguration do
+  describe 'accessors' do
+    it 'should have accessors for the needed attributes' do
```

Comment:
The test is just to ensure the methods are defined. It doesn't check where
they actually work. I thought it was interesting, since it is the defining
part of the change, and defines the API.

The have_attr_accessor matcher unfortunately isn't part of rspec, but I
imported it from a StackOverflow answer.

Em sáb, 4 de jun de 2016 21:57, Eduardo Silva <notifications@github.com>
escreveu:

> In spec/entities/configurations/kalibro_configuration_spec.rb
> <https://github.com/mezuro/kalibro_client/pull/110#discussion_r65809835>:
>
> > @@ -17,6 +17,14 @@
> >  require 'spec_helper'
> >
> >  describe KalibroClient::Entities::Configurations::KalibroConfiguration do
> > +  describe 'accessors' do
> > +    it 'should have accessors for the needed attributes' do
>
> It does :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mezuro/kalibro_client/pull/110/files/70427ae757120cfd73f7e51077190a42b8010cfe#r65809835>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAtnTnZwp_Rica8ZCWBvV5DK7fOtqWP1ks5qIh7igaJpZM4IqDsX>
> .
>